### ISSUE TITLE
Whitespace after line continuation breaks syntax highlighting

### ISSUE THREAD

**Issue by eortega-pjr** [camdencheek/tree-sitter-dockerfile#26]
sorry if this is already known
```
RUN echo "foo" \<space> 
    echo "feh"
### no syntax hereafter due to space after \ above
```

**Comment by zbroniszewski** [comment]
Would have never caught that!

Glad I found this.

**Comment by camdencheek** [comment]
It's actually pretty surprising to me that docker accepts that. In most other languages I've used, the backslash is escaping only the next character, which is not a newline in this case. In any case, fixed it in #49

**Comment by eortega-pjr** [comment]
> It's actually pretty surprising to me that docker accepts that. In most other languages I've used, the backslash is escaping only the next character, which is not a newline in this case. In any case, fixed it in #49

I wouldn't be surprised if the Dockerfile parser trims whitespace off the end of each line.
